### PR TITLE
add script to compare WHO to OWID and save results to S3

### PR DESCRIPTION
**Pull request by muamichali**

### CODE REVIEW

File: scripts/compare-globalvax.py [hunk 1]
```diff
@@ -0,0 +1,165 @@
+""" Script that runs through validated manual screenshots in Airtable and uploads them to S3. """
```
Comment (gilmourj) [inline scripts/compare-globalvax.py]: let's change the header to reflect what we're doing here, like "Script that compares WHO and OWID total vaccine doses and uploads a file with results to S3."

File: scripts/compare-globalvax.py [hunk 2]
```diff
@@ -0,0 +1,165 @@
...
+    default='pandemic-tracking-collective-data',
+    help='S3 bucket name')
+
+parser.add_argument(
+    '--s3-subfolder',
+    default='globalvax',
+    help='Name of subfolder on S3 bucket to upload files to')
+
+parser.add_argument('--push-to-s3', dest='push_to_s3', action='store_true', default=False,
+    help='Push screenshots to S3')
```
Comment (gilmourj) [inline scripts/compare-globalvax.py]: maybe something like "Push data to S3"?

**Review by gilmourj — CHANGES_REQUESTED** [review]
Looks great! I think we should just change the language to reflect what we're doing here, but otherwise good to go. Thank you!

File: scripts/compare-globalvax.py (hunk 2 above)
Comment (muamichali) [reply scripts/compare-globalvax.py]: ty, this makes sense. Addressed below!

**Review by gilmourj — APPROVED** [review]

**Pull request merged**